apps/www/_blog/2023-08-12-vecs-adapters.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Vecs Adapters'
+description: 'supabase/vecs adapters: query vectors with text'
+author: oli_rice
+image: 2023-06-01-vecs/og-image.jpg
+thumb: 2023-06-01-vecs/og-image.jpg
+tags:
+  - postgres
+  - AI
+  - python
+date: '2023-08-11'
+toc_depth: 3
+---
+
+[supabase/vecs](https://github.com/supabase/vecs) is our python client for rapid development of vector projects based on postgres + pgvector.
+
+Today, we're releasing a new feature for vecs called "adapters" that enables developers to manage embeddings without ever having to touch a vector.
```

Comment:
> "without ever having to touch a vector."

I'm wondering if they ever had to touch a vector previously? Perhaps, "without ever having to touch an external lib" (needs rephrasing, but hopefully makes the benefit clearer)

Reply:
> I'm wondering if they ever had to touch a vector previously?

yes, without adapters the user was responsible for encoding the vector and then passing the vector to `vecs`